daffodil-test/src/test/scala/org/apache/daffodil/extensions/TestInputTypeValueCalc.scala

```diff
@@ -43,10 +43,12 @@ class TestInputTypeValueCalc {
   @Test def test_InputTypeCalc_keysetValue_00(): Unit = { runner.runOneTest("InputTypeCalc_keysetValue_00") }
   @Test def test_InputTypeCalc_keysetValue_01(): Unit = { runner.runOneTest("InputTypeCalc_keysetValue_01") }
   @Test def test_InputTypeCalc_keysetValue_02(): Unit = { runner.runOneTest("InputTypeCalc_keysetValue_02") }
+  @Test def test_InputTypeCalc_keysetValue_03(): Unit = { runner.runOneTest("InputTypeCalc_keysetValue_03") }
```

Comment:
The name of this test probably should change also, as I don't think of this issue as a "inputTypeCalc" nor "keysetValue" thing. Suggestion: test_repTypeOnTypeRequiresLengthKindImplicit